### PR TITLE
Adds some gremlin examples and notes.

### PR DESCRIPTION
**Pull request by jtudelag**
* HasKey
* Regex

### CODE REVIEW

**Comment by jtudelag** [comment]
@lebauce @safchain PTAL ;

File: documentation/api-gremlin.md [hunk 1]
```diff
@@ -452,8 +468,8 @@ G.V().Has('Type', Without('ovsbridge', 'ovsport'))
   expression.
 
 {% highlight shell %}
-G.V().Has('Name', Regex('tap.*'))
+G.V().Has('Name', Regex("tap.*"))
```
Comment (safchain) [inline documentation/api-gremlin.md]: mm, single quote is supported maybe your issue comes from the mix of single quote/double quote in the command line

Comment (jtudelag) [reply documentation/api-gremlin.md]: @safchain you are right, I tried it, and simple quotes and they just work as well. Removed from the PR.

**Comment by safchain** [comment]
thanks for the PR :)

**Pull request merged**